src/Elcodi/Component/Banner/Tests/UnitTest/Services/BannerManagerTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014-2015 Elcodi Networks S.L.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Component\Banner\Tests\UnitTest\Services;
+
+use Elcodi\Component\Banner\Repository\BannerRepository;
+use Elcodi\Component\Banner\Services\BannerManager;
+use PHPUnit_Framework_TestCase;
+
+class BannerManagerTest extends PHPUnit_Framework_TestCase
```

Comment:
It's a classic unit test, just to test that `BannerManager` continue to have the same specification.
For example: `getBannersFromBannerZoneCode` shouldn't throw an exception, if it does (with a random string) than there's a bug.